app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -99,6 +99,7 @@ def index_relation_product(query)
     def index_relation_content_view(query)
       if params[:content_view_version_id]
         query = query.where(:content_view_version_id => params[:content_view_version_id])
+        query = query.archived unless params[:library] || params[:environment_id]
```

Comment:
I guess what i'm asking is should we even ever return the non-archived repos for a Content view version?